spec/features/vote_spec.rb

```diff
@@ -53,9 +53,26 @@
 } do
   let!(:review) { FactoryGirl.create(:review) }
 
-  scenario "visitor tries to down vote on review redirects to signin" do
+  scenario "visitor can't vote on review" do
     visit restaurant_path(review.restaurant)
-    click_button("Down-vote")
-    expect(page).to have_content("Log in")
+
+    expect(page).not_to have_button("Up-vote")
+    expect(page).not_to have_button("Down-vote")
+  end
+end
+
+feature "visitor cannot vote on a review", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.